src/controllers/favorites.js

```diff
@@ -108,6 +108,15 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
             overlayMoreButton: !0,
             action: "instantmix",
             coverImage: !0
+        }, {
+            name: "HeaderFavoriteBooks",
+            types: "Book",
+            shape: getPosterShape(),
+            showTitle: !0,
```

Comment:
We should use true/false for these. The others are only using this format because the source was originally uglified.

Reply:
@thornbill does it make sense to leave this for a separate clean up effort? I could do the entire file but it would make sense to do it as a separate effort to have the change under a different merge.

Reply:
I think @thornbill just means the small section you added should use boolean values, we definitely don't expect you to change all of them in this pull request.

Reply:
Ah right. I meant just in the section you added. Thanks for clarifying @dkanada!